src/highdicom/sr/content.py

```diff
@@ -707,6 +703,57 @@ def from_dataset(
         return cast(SourceSeriesForSegmentation, item)
 
 
+class CoordinatesForMeasurement(ScoordContentItem):
+
+    """Content item representing spatial coordinates of a measurement"""
+
+    def __init__(
+            self,
+            graphic_type: Union[GraphicTypeValues, str],
+            graphic_data: np.ndarray,
+            source_image: SourceImageForRegion,
+            purpose: Union[CodedConcept, Code] = codes.SCT.Source,
+    ) -> None:
+        graphic_type = GraphicTypeValues(graphic_type)
+        super().__init__(
+            name=purpose,
+            graphic_type=graphic_type,
+            graphic_data=graphic_data,
+            relationship_type=RelationshipTypeValues.INFERRED_FROM,
+        )
+        self.ContentSequence = [source_image]
+
+    @classmethod
+    def from_dataset(
+            cls,
+            dataset: Dataset,
+            copy: bool = True,
+    ) -> 'CoordinatesForMeasurement':
+        """Construct object from an existing dataset.
+
+        Parameters
+        ----------
+        dataset: pydicom.dataset.Dataset
+            Dataset representing an SR Content Item with value type SCOORD
+        copy: bool
+            If True, the underlying dataset is deep-copied such that the
+            original dataset remains intact. If False, this operation will
+            alter the original dataset in place.
+
+        Returns
+        -------
+        highdicom.sr.CoordinatesForMeasurement
+            Constructed object
+
+        """
+        if copy:
+            dataset_copy = deepcopy(dataset)
+        else:
+            dataset_copy = dataset
+        item = super()._from_dataset_base(dataset_copy)
+        return cast(CoordinatesForMeasurement, item)
```

Comment:
Going forward I'm doing this for better inheritance behaviour. If I remember correctly, a change in the `0.24.0` dev branch makes this change for all such classes.

```suggestion
        return cast(cls, item)
```

There's also the matter of then how to correctly type the return type in this case. But don't worry I'll make that change before release.